src/main/java/thePackmaster/cards/AbstractPackmasterCard.java

```diff
@@ -82,18 +82,7 @@ protected Texture getPortraitImage() {
     }
 
     public static String getCardTextureString(final String cardName, final AbstractCard.CardType cardType) {
-        String textureString;
-
-        switch (cardType) {
-            case ATTACK:
-            case POWER:
-            case SKILL:
-                textureString = makeImagePath("cards/" + cardName + ".png");
-                break;
-            default:
-                textureString = makeImagePath("ui/missing.png");
-                break;
-        }
+        String textureString = makeImagePath("cards/" + cardName + ".png");
```

Comment:
This is the fix I mentioned to you. There's no reason to have a switch on cardtype here at all.